pkg/solana/chain.go

```diff
@@ -557,21 +549,25 @@ func (c *chain) sendTx(ctx context.Context, from, to string, amount *big.Int, ba
 				toKey,
 			).Build(),
 		},
-		blockhash.Value.Blockhash,
```

Comment:
I think it's ok if the caller to `Enqueue` sets a block hash here. We'll just overwrite it when needed.